frontend/appflowy_flutter/windows/runner/main.cpp

```diff
@@ -5,6 +5,9 @@
 #include "flutter_window.h"
 #include "utils.h"
 
+#include <bitsdojo_window_windows/bitsdojo_window_plugin.h>
+auto bdw = bitsdojo_window_configure(BDW_CUSTOM_FRAME | BDW_HIDE_ON_STARTUP);
```

Comment:
This is the deciding factor on the frame for the window @LucasXu0

Reply:
Nice. I tested it, and it works!